cmd/influx/delete.go

```diff
@@ -33,7 +33,7 @@ func newDeleteCmd() cli.Command {
 				Value:  &params.BucketID,
 			},
 			&cli.StringFlag{
-				Name:        "bucket",
+				Name:        "bucket, b",
```

Comment:
Can you add a note to the CHANGELOG.md for this? We haven't automated it here yet